### ISSUE TITLE
Option to delete site-cookies

### ISSUE THREAD

**Issue by Micha-he** [LakorTi/Via#433]
Can we get an option to delete the cookies of the currently open website? Or does it already exist?

**Comment by LakorTi** [comment]
Why do you have this request? Log out of a specific website?

**Comment by Micha-he** [comment]
Yes. Sometimes I've websites where I   logged in, but I receive a error, that I should log in. When I try to logout, I receive also an error.

The only option is: Delete all cached cockies and log in to many websites.

**Comment by tuyafeng** [comment]
Sounds terrible, I will consider adding this option to the application, probably in the next two versions.

**Comment by Micha-he** [comment]
My ideas: Either 'Delete all cookies from this site' under tools or a site-filter-string at settings-delete data-cookies, so we can limit the delete command to some server/domains.

**Comment by tuyafeng** [comment]
I think it might be placed in the "Site conf" pop-up.

**Comment by zhangjing-GitHub-Code** [comment]
Hey, seems like the feature is added, but it is COMPLETELY USELESS, IT HAS NO EFFECT AT ALL. I CAN ALWAYS see BUNCH OF COOKIES on login.microsoftonline.com NO MATTER HOW MANY DELETIONS I MADE. And I CAN'T SEE ANY COOKIE in script cookie editors.
@solokot
@gregory678
@kae-sakura
@betterwebleon
@tuyafeng
